vaadin-date-picker-flow-parent/vaadin-date-picker-flow/src/main/resources/META-INF/resources/frontend/datepickerConnector.js

```diff
@@ -64,25 +65,71 @@ import dateFnsIsValid from 'date-fns/isValid';
             throw new Error('Array of custom date formats is null or empty');
           }
 
+          function _shortenFourDigitYearFormat(format) {
```

Comment:
```suggestion
          function getShortYearFormat(format) {
```

Terminology could be improved to always use the same term like short year. No need to prefix any of the newly added functions with an underscore.